defold-spine/commonsrc/vertices.cpp

```diff
@@ -38,11 +42,20 @@ static uint32_t EnsureArrayFitsNumber(dmArray<T>& array, uint32_t num_to_add)
     return prev_size;
 }
 
+template <typename T>
+static void EnsureArraySize(dmArray<T>& array, uint32_t size)
+{
+    if (array.Capacity() < size)
+    {
+        array.SetCapacity(size);
+    }
+    array.SetSize(size);
+}
```

Comment:
I kinda feel this is a useful function to have in dmsdk, I see variants of this (and the one above) now and then.. Maybe some sort of array utility header in dmsdk?